app/src/main/java/com/lagradost/cloudstream3/ui/player/GeneratorPlayer.kt

```diff
@@ -1095,7 +1095,8 @@ class GeneratorPlayer : FullScreenPlayer() {
         }
 
         playerBinding?.playerSkipOp?.isVisible = isOpVisible
-        playerBinding?.playerSkipEpisode?.isVisible =
+
+        playerBinding?.playerGoForward?.isVisible =
```

Comment:
`playerGoForward is visible from the beginning not after 50% as it depends on hasNextEpisode not PreloadNextEpisode` 
No? `isOpVisible = percentage < SKIP_OP_VIDEO_PERCENTAGE` => `playerGoForward.isVisible = percentage > SKIP_OP_VIDEO_PERCENTAGE` meaning that in the event on an anime, the top button is not visible on TV. This was the behavior before on phone as we did not want to clutter the bottom bar, but on TV this should not be the behavior.